app/utils/draft.js

```diff
@@ -109,14 +109,18 @@ export function alertSlashCommandFailed(formatMessage, error) {
     );
 }
 
-export function buildChannelWideMentionMessage(formatMessage, membersCount, isTimezoneEnabled, channelTimezoneCount) {
+export function buildChannelWideMentionMessage(formatMessage, membersCount, isTimezoneEnabled, channelTimezoneCount, atHere) {
     let notifyAllMessage = '';
     if (isTimezoneEnabled && channelTimezoneCount) {
+        const msgID = atHere ? t('mobile.post_textbox.entire_channel_here.message.with_timezones') : t('mobile.post_textbox.entire_channel.message.with_timezones');
+        const atHereMsg = 'By using @here you are about to send notifications up to {totalMembers, number} {totalMembers, plural, one {person} other {people}} in {timezones, number} {timezones, plural, one {timezone} other {timezones}}. Are you sure you want to do this?';
```

Comment:
is this number taken into account only those people that are "online"? notifications for `@here` is only on the UI and desktop notifications iirc

Reply:
I am not changing numbers here. We are just rephrasing a bit the message so that it reads "up to N people" so that we take into account the fact that they may be less.